modules/publicize/publicize.php

```diff
@@ -418,15 +419,99 @@ function save_meta( $post_id, $post ) {
 		// Next up will be ::publicize_post()
 	}
 
+	public function update_published_message( $messages ) {
+		global $post_type, $post_type_object, $post;
+		if ( ! $this->post_type_is_publicizeable( $post_type ) ) {
+			return $messages;
+		}
+		$view_post_link_html = '';
+		$viewable = is_post_type_viewable( $post_type_object );
+		if ( $viewable ) {
+			$view_text = esc_html__( 'View post' ); // intentionally omitted domain
```

Comment:
@enejb do you remember why was domain omitted here?

I ran `yarn add-textdomain` (a script that adds textdomains to php files where they are missing) and it caught up on this one.